docs/source/API/core/policies/RangePolicy.rst

```diff
@@ -141,6 +141,7 @@ Preconditions:
 
 * The start index must not be greater than the end index.
 * The actual constructors are templated so we can check that they are converted to ``index_type`` safely (see `#6754 <https://github.com/kokkos/kokkos/pull/6754>`_).
+   * The conversion safety check is only performed if ``index_type`` is convertible to the start and end index types.
```

Comment:
start -> begin for consistency

Reply:
Looks like there were already mixed usages of start/end and begin/end throughout this document.
Changed all `start` to `begin`.  6cd5e99